src/redux/books/books.js

```diff
@@ -0,0 +1,31 @@
+// Actions
+
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+// Reducer
```

Comment:
- [ ] Kindly Sets an initial state with an empty array eg: `const initialState = [ ];`

Reply:
Hello @BenMukebo, I have a question.
https://github.com/GiuseppeTG/Bookstore/blob/8ddf262958de347c4b5211dcd01368c0158cf628/src/redux/books/books.js#L8
I understood that by using the parameter state = [] is the shorthand of ES6 of doing what you are telling me. So, can you please explain me why is that change required?

Reply:
Oh! It is just that it will be useful for the next activity... no problem then.

Reply:
Great then 👍🏻